pkg/scheduler/cache/cache.go

```diff
@@ -161,7 +162,14 @@ func (db *DefaultBinder) Bind(kubeClient kubernetes.Interface, tasks []*scheduli
 	var errTasks []*schedulingapi.TaskInfo
 	for _, task := range tasks {
 		p := task.Pod
-		if err := kubeClient.CoreV1().Pods(p.Namespace).Bind(context.TODO(),
+		if p.Annotations["is-fake"] == "true" {
```

Comment:
I'm sorry for not get the usage of `is-fake` annotation, can you add more description about this PR?